input/haleu/sensitivity-analysis/oat/mmr_burnup/dakota.in

```diff
@@ -8,14 +8,19 @@ environment,
 	write_restart 'mmr_burnup.rst'
 
 method,						
-	multidim_parameter_study
-    partitions = 10
+	list_parameter_study
+          list_of_points = 41
+                           62
+                           74
+                           78
+                           82
+                           86
+                           90
+                          
```

Comment:
For the other variations (transition start time, lwr lifetimes, and build shares) I am sweeping over a grid of evenly spaced points. However, the points I want to use for the burnup variations (applied to the MMR or Xe-100) are not evenly spaced. So I am telling Dakota to use these specific values instead of an even grid like I do when varying the parameters.